### DISCUSSION TITLE
安装提示缺少库文件

### DISCUSSION THREAD

**Discussion by Raphel1** [kubegems/kubegems#118] [Q&A]
安装kubegems的时候提示缺少Error: Error 1146: Table 'gemcloud.tenants' doesn't exist

**Comment by pepesi** [comment]
等待job kubegems-init-main 跑完就行了

**Reply by Raphel1** [reply]
不行，都跑了六个小时了
![图片](https://user-images.githubusercontent.com/71837156/174257405-8dc2c870-446c-4773-ab1b-5802d8d14822.png)

**Reply by Raphel1** [reply]
我安装的k8s版本是v21.2

**Reply by Raphel1** [reply]
而且这个库是我自己创建的gemcloud

**Comment by pepesi** [comment]
kubectl get job 看看呢，似乎没有跑起来init的job

**Reply by Raphel1** [reply]
![图片](https://user-images.githubusercontent.com/71837156/174258048-c7d4db31-21cf-4537-8993-0c9682e3b2e1.png)

**Reply by Raphel1** [reply]
我这么让job重新跑一次呢
![图片](https://user-images.githubusercontent.com/71837156/174260404-57d17a2c-ac37-4af5-9894-51b1fe853792.png)

**Reply by pepesi** [reply]
kubectl logs gems-mysql-initialize-v1.20.0-beta.2

**Reply by Raphel1** [reply]
没有这个pod

**Reply by pepesi** [reply]
重新跑一次job吧

**Reply by Raphel1** [reply]
这么重新启动呢，我没有找到运行这个的yaml

**Reply by pepesi** [reply]
kubectl get job gems-mysql-initialize-v1.20.0-beta.2

**Reply by Raphel1** [reply]
我已经把这个job kubectl delete删除了，这么能重新创建这个job

**Reply by Raphel1** [reply]
初始化这个pod报错了
![图片](https://user-images.githubusercontent.com/71837156/174263838-1df10d8b-d038-4c09-8aca-2ba582531a38.png)

**Reply by pepesi** [reply]
哦，你把自己创建的database删除了吧。job会自己创建的，似乎这个是字符集不对导致的

**Reply by Raphel1** [reply]
好的，谢谢，已经好了

**Reply by Raphel1** [reply]
你好，页面报404了
![图片](https://user-images.githubusercontent.com/71837156/174272572-c53401c2-cf51-4e76-a595-ff477fd1b5eb.png)
我看配置文件写的是/,是不是访问需要添加路径
![图片](https://user-images.githubusercontent.com/71837156/174272801-3e88cc6e-3cc4-4f82-856e-020393a5a0a3.png)

**Reply by Raphel1** [reply]
使用nodeport无法访问
